lib/DataGrid.php

```diff
@@ -55,62 +48,42 @@ class DataGrid implements DataGridInterface
 
     public function __construct(
         string $name,
-        DataGridFactoryInterface $dataGridFactory,
-        DataMapperInterface $dataMapper
+        DataGridFactoryInterface $dataGridFactory
     ) {
         $this->name = $name;
         $this->dataGridFactory = $dataGridFactory;
-        $this->dataMapper = $dataMapper;
         $this->eventDispatcher = new EventDispatcher();
```

Comment:
This could also be passed into the `__construct()` as an optional value and only be initiated in case of the parameter being a `null`. That way we could pass the event dispatcher from Symfony's container.